src/Servers/IIS/IIS/test/testassets/InProcessWebSite/ThrowingStartup.cs

```diff
@@ -9,11 +9,6 @@ namespace TestSite
 {
     public class ThrowingStartup
     {
-        public void ConfigureServices(IServiceCollection serviceCollection)
```

Comment:
I think you want different tests. This fails at a different time in each of the hosting scenarios. (Web vs Generic host)

Reply:
I disagree, I think it's redundant and increases test time for a poor reason. Each of these tests increase CI validation time by ~4 seconds.